src/envelopeCommands.cpp

```diff
@@ -184,23 +185,24 @@ optional<int> currentEnvelopePoint{};
 
 const char* getEnvelopeShapeName(int shape) {
 	static const char* names[] = {
-		// Translators: A shape for an envelope point.
-		translate("linear"),
-		// Translators: A shape for an envelope point.
-		translate("square"),
-		// Translators: A shape for an envelope point.
-		translate("slow start/end"),
-		// Translators: A shape for an envelope point.
-		translate("fast start"),
-		// Translators: A shape for an envelope point.
-		translate("fast end"),
-		// Translators: A shape for an envelope point.
-		translate("bezier"),
-	};
+			// Translators: A shape for an envelope point.
+			translate("linear"),
+					// Translators: A shape for an envelope point.
+					translate("square"),
+							// Translators: A shape for an envelope point.
+							translate("slow start/end"),
```

Comment:
Wo. What on earth is going on here? It seems to keep adding continuation indents even though each translate call is closed. I wonder if WhitespaceSensitiveMacros is interfering here somehow?

Reply:
It does seem that AlignAfterOpenBracket: DontAlign makes this worse. But it's definitely very confused. If I add random indentation to these lines, it preserves that even with AlignAfterOpenBracket: BlockIndent. So I guess the whitespace sensitivity applies to the line before the macro, not just inside the macro, which is very unfortunate.

I think I'm going to need to fix makePot... somehow.

Reply:
I'm not sure. There's also some black magic going on in uia.cpp I referenced earlier, which I think is related. It really looks like we are missing an alignment setting somewhere, but I'm pretty sure we have covered them all.

I'm starting to regret this all.

Reply:
I really do appreciate you working on this. Even if we end up shelving it for a while, we can always come back to it.

I'll look at fixing makePot at some point to cope with multi-line calls. I have some ideas on how it might be done and that's probably a good idea for other reasons anyway.

Losing translateFirstString is a bit of a shame, but I guess that was always a bit sketchy anyway, and not having to worry about code formatting eventually (which is a total waste of people's time) probably outweighs that loss.

Reply:
Sure. I will mark this as draft for now.